catalogs/catalog-hadoop/src/test/java/com/datastrato/gravitino/catalog/hadoop/integration/test/HadoopCatalogIT.java

```diff
@@ -197,6 +197,31 @@ public void testCreateFileset() throws IOException {
         Fileset.Type.MANAGED, fileset3.type(), "fileset type should be MANAGED by default");
   }
 
+  @Test
+  public void testCreateFilesetWithChineseComment() throws IOException {
+    // create fileset
+    String filesetName = "test_create_fileset";
+    String storageLocation = storageLocation(filesetName);
+    Assertions.assertFalse(
+        hdfs.exists(new Path(storageLocation)), "storage location should not exists");
+    Fileset fileset =
+        createFileset(
+            filesetName,
+            "这是中文comment",
+            Fileset.Type.MANAGED,
+            storageLocation,
+            ImmutableMap.of("k1", "v1"));
```

Comment:
I think it is not just a comment that is affected by this issue, but also like storageLocation, property key and value, right? Can you also add tests to cover this?

Reply:
Add some assertions for this.